repo.sh

```diff
@@ -20,83 +20,79 @@ fi
+    ["https://github.com/openedx/edx-analytics-dashboard.git"]="open-release/olive.master"
+    ["https://github.com/openedx/edx-analytics-data-api.git"]="open-release/olive.master"
+    ["https://github.com/openedx/frontend-app-authn.git"]="open-release/olive.master"
+    ["https://github.com/openedx/frontend-app-course-authoring.git"]="open-release/olive.master"
+    ["https://github.com/openedx/frontend-app-learning.git"]="open-release/olive.master"
+    ["https://github.com/openedx/frontend-app-library-authoring.git"]="master"
+    ["https://github.com/openedx/registrar.git"]="master"
+    ["https://github.com/openedx/frontend-app-program-console.git"]="master"
+    ["https://github.com/openedx/frontend-app-account.git"]="open-release/olive.master"
+    ["https://github.com/openedx/frontend-app-profile.git"]="open-release/olive.master"
+    ["https://github.com/edx/frontend-app-ora-grading.git"]="open-release/olive.master"
 )
 
-non_release_repos=(
-    "https://github.com/openedx/frontend-app-authn.git"
-    "https://github.com/openedx/frontend-app-course-authoring.git"
-    "https://github.com/openedx/frontend-app-learning.git"
-    "https://github.com/openedx/frontend-app-library-authoring.git"
-    "https://github.com/openedx/registrar.git"
-    "https://github.com/openedx/frontend-app-program-console.git"
-    "https://github.com/openedx/frontend-app-account.git"
-    "https://github.com/openedx/frontend-app-profile.git"
-    "https://github.com/edx/frontend-app-ora-grading.git"
+declare -A ssh_repos=(
+    ["git@github.com:openedx/course-discovery.git"]="open-release/olive.master"
+    ["git@github.com:openedx/credentials.git"]="open-release/olive.master"
+    ["git@github.com:openedx/cs_comments_service.git"]="open-release/olive.master"
+    ["git@github.com:openedx/ecommerce.git"]="open-release/olive.master"
+    ["git@github.com:edx/edx-e2e-tests.git"]="master"
+    ["git@github.com:openedx/edx-notes-api.git"]="open-release/olive.master"
+    ["git@github.com:Pearson-Advance/edx-platform.git"]="pearson-release/olive.main"
+    ["git@github.com:openedx/xqueue.git"]="open-release/olive.master"
+    ["git@github.com:openedx/edx-analytics-dashboard.git"]="open-release/olive.master"
+    ["git@github.com:openedx/frontend-app-gradebook.git"]="open-release/olive.master"
+    ["git@github.com:openedx/frontend-app-payment.git"]="open-release/olive.master"
+    ["git@github.com:openedx/frontend-app-publisher.git"]="open-release/olive.master"
+    ["git@github.com:openedx/edx-analytics-dashboard.git"]="open-release/olive.master"
+    ["git@github.com:openedx/edx-analytics-data-api.git"]="open-release/olive.master"
+    ["git@github.com:openedx/frontend-app-authn.git"]="open-release/olive.master"
+    ["git@github.com:openedx/frontend-app-course-authoring.git"]="open-release/olive.master"
+    ["git@github.com:openedx/frontend-app-learning.git"]="open-release/olive.master"
```

Comment:
Shouldn't it be our Learning MFE?. In case we need to use our Learning MFE, then we need to also setup the sidebar MFE. @alexjmpb